### PR TITLE
(SIMP-2755) Finished adding reference to readme

### PR DESCRIPTION
**Pull request by cmentzer**
Changes according to discussion of readme style with Kendall, Ryan, Tyler, and others.

SIMP-2755 #close

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,22 +63,519 @@ include '::auditd'
...
-```yaml
----
 auditd::at_boot : false
+
+#### Changing Key Values
+```puppet
+# To override the default values included in the module, you can either
+# include new values for the keys at the time that the classes are declared:
+
+class { 'auditd':
```
Comment (jeefberkey) [inline README.md]: This code is invalid

File: README.md [hunk 2]
```diff
@@ -74,22 +63,519 @@ include '::auditd'
...
+  * Default: 'SUSPEND'
+
+##### `disk_error_action`:
+
+  * Valid Options: Auditd::DiskErrorAction
+  * Default: 'SUSPEND'
+
+##### `default_audit_profile`:
+
+  * Valid Types: 'simp', Boolean]
```
Comment (jeefberkey) [inline README.md]: Trailing ]

File: README.md [hunk 3]
```diff
@@ -74,22 +63,519 @@ include '::auditd'
...
+
+##### `priority`:
+
+  * Valid Options: Auditd::LogPriority
+  * Default:
+
+##### `facility`:
+
+  'LOG_INFO'
+  * Valid Options: Auditd::LogFacility
```
Comment (jeefberkey) [inline README.md]: Should we expand this type alias?

Comment (cmentzer) [inline README.md]: Dunno, was something that came up in our discussion and never reached a good conclusion. It's kind of slippery to make expanding them standard.

File: README.md (hunk 1 above)
Comment (cmentzer) [inline README.md]: R I P lemme fix that

File: README.md (hunk 2 above)
Comment (cmentzer) [inline README.md]: Fixed

File: README.md (hunk 3 above)
Comment (jeefberkey) [inline README.md]: I'd have to look at code to find out what a Auditd::LogFacility is, so it's kinda useless here.

File: README.md [hunk 4]
```diff
@@ -74,22 +63,519 @@ include '::auditd'
...
----
 auditd::at_boot : false
+
+#### Changing Key Values
+```puppet
+# To override the default values included in the module, you can either
+# include new values for the keys at the time that the classes are declared:
+
+class { '::auditd':
+  ignore_failures => true
```
Comment (jeefberkey) [inline README.md]: Needs commas

File: README.md [hunk 5]
```diff
@@ -74,22 +63,519 @@ include '::auditd'
...
+
+  When set to true, ensures rules are added absolutely first or last depending on $first
+  * Valid Options: Boolean
+  * Default: false
+
+##### `content`:
+
+  The content of the rule to be added
+  * Valid Options: String
+  * Default: N/A
```
Comment (jeefberkey) [inline README.md]: undef

Comment (cmentzer) [inline README.md]: Updated

File: README.md (hunk 4 above)
Comment (cmentzer) [inline README.md]: Added commas

**Pull request merged**